packages/grafana-perf-tests/webpack.config.js

```diff
@@ -0,0 +1,50 @@
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const CopyPlugin = require('copy-webpack-plugin');
+const path = require('path');
+const GlobEntries = require('webpack-glob-entries');
+
+module.exports = {
+  mode: 'production',
+  entry: GlobEntries('./src/*test*.ts'), // Generates multiple entry for each test
+  output: {
+    path: path.join(__dirname, 'dist'),
+    libraryTarget: 'commonjs',
+    filename: '[name].js',
+  },
+  resolve: {
+    extensions: ['.ts', '.js'],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.ts$/,
+        use: 'babel-loader',
+        exclude: /node_modules/,
+      },
+    ],
+  },
+  target: 'web',
+  externals: /^(k6|https?\:\/\/)(\/.*)?/,
+  // Generate map files for compiled scripts
+  devtool: 'source-map',
+  stats: {
+    colors: true,
+  },
+  plugins: [
+    new CleanWebpackPlugin(),
```

Comment:
Do we need this dependency? Webpack has a built in [clean option](https://webpack.js.org/guides/output-management/#cleaning-up-the-dist-folder).

Reply:
I copied the [k6 typescript template](https://github.com/grafana/k6-template-typescript) - I'll try to change this

Reply:
remoed